CHANGELOG.md

```diff
@@ -907,6 +907,7 @@
 ## [unreleased]
 
 - Add Ghost Inspector tests to the GitHub action deploy workflow
+- Fix Ghost Inspector action (use suite instead of test)
```

Comment:
probably this can stay as "Add Ghost Inspector tests to the GitHub action deploy workflow" in terms of release notes?

Reply:
Okie yup, I'll remove that then